### ISSUE TITLE
issue on fresh install on Mac with HomeBrew, unable to register remote plugins

### ISSUE THREAD

**Issue by bigfish** [neovim/node-host#9]
I've installed neovim homebrew
![screen shot 2016-01-13 at 4 54 42 pm](https://cloud.githubusercontent.com/assets/34862/12309023/6703d972-ba16-11e5-8f70-abec81019312.png)

I've run npm install in node-host plugin /node repo, and in my plugin repos, but keep getting this error:

![screen shot 2016-01-13 at 4 56 01 pm](https://cloud.githubusercontent.com/assets/34862/12309062/8f9cc042-ba16-11e5-95c6-c69bb8db32a3.png)

running node 5.4.1 using nvm. Tried downloaded pkg version from node website but also same issue.
Any help appreciated, I'd love to get the plugins working on the mac. I occasionally saw this on Ubuntu as well, but it seemed to go away after a while.

**Comment by nhynes** [comment]
Interesting. Could you please post the contents of your `~/.nvimlog`? It might also help if you do `export NEOVIM_JS_DEBUG=/tmp/nvim_js_debug` and then post the contents of that, too.

**Comment by bigfish** [comment]
I don't have any ~/.nvimlog. I set the NEOVIM_JS_DEBUG to ~/.nvjslog but I don't see it either. Do I need to have a different build or set some startup option to enable debugging?

**Comment by nhynes** [comment]
> I don't have any ~/.nvimlog

Yep, sorry. I didn't catch that you were running the release build. Can you try, at least temporarily, [installing neovim from HEAD](https://github.com/neovim/homebrew-neovim/blob/master/README.md)? 

I think that the error is being caused by one of your plugins. You might also want to register each one individually (possibly by shuffling files around) to see which, if any, causes the error.

**Comment by bigfish** [comment]
ok, I installed the HEAD version using homebrew. I now get .nvimlog. 

when I restarted it complained about 

2016/01/13 20:12:37 [info @ main:523] 33384 - Starting Neovim main loop.
2016/01/13 20:12:55 [error @ forward_stderr:333] 33384 - Channel 1 stderr: events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '~/.nvjslog'
    at Error (native)

---

Then I realized the real reason was that in my .bashrc I had quotes around the path, so the ~ wasn't being expanded I guess. 

Anyways, I'll close the issue now, thanks.
